services/core-web/src/tests/components/mine/ExplosivesPermit/ExplosivesPermit.spec.js

```diff
@@ -5,12 +5,6 @@ import * as MOCK from "@/tests/mocks/dataMocks";
 
 const props = {};
 
-jest.mock("@mds/common/providers/featureFlags/useFeatureFlag", () => ({
```

Comment:
Must have added it here...I think I was tired and it didn't work immediately when I tried to move it into the setupTests... 😁